Error while running "yarn run build"

On running `yarn run build` I am getting this issue 
```

ERROR in ./src/js/reducers/index.js
Module not found: Error: Can't resolve './DomReducer' in '/home/girish/Github/trezor/connect-explorer/src/js/reducers'
 @ ./src/js/reducers/index.js 9:18-41
 @ ./src/js/store/index.js
 @ ./src/js/index.js
 @ multi ./src/js/index.js

```


## Machine specification
Nodejs version - 10.13.0
npm version - 6.4.1
OS - Ubuntu 18.04
Kernel - 4.15.0-36-generic

--- a/src/js/reducers/index.js
+++ b/src/js/reducers/index.js
@@ -4,7 +4,7 @@
 import { combineReducers } from 'redux';
 import { routerReducer } from 'react-router-redux';
 
-import DOM from './DomReducer';
+import DOM from './DOMReducer';
 import connect from './TrezorConnectReducer';
 import modal from './ModalReducer';
 

